task.go

```diff
@@ -0,0 +1,18 @@
+package libcarina
+
+type Input interface{}
```

Comment:
_Kyle whispers_ "You'll save yourself a lot of long term pain if you can strongly type this and not use `interface{}`"